client/src/pages/Report/ApplicationSection/ApplicationSection.tsx

```diff
@@ -40,6 +40,29 @@ const ApplicationSection: React.FC<ApplicationSectionProps> = ({
     const [showCustomPromptPopup, setShowCustomPromptPopup] = useState(false);
     const [selectedApp, setSelectedApp] = useState<ApplicationDataRow | null>(null);
     const [selectedApplications, setSelectedApplications] = useState<ApplicationDataRow[]>([]);
+    const [applicationsToAdd, setApplicationsToAdd] = useState<ApplicationDataRow[]>([]);
+
+    const loadApplications = async () => {
+        try {
+            const data = await ManagmentServiceApiInstance.getApplications(clusterId);
+            setApplicationsToAdd(
+              data.map((app) => ({
+                  name: app.name,
+                  running: app.running,
+                  kind: app.kind,
+                  accuracy: defaultAccuracy,
+                  customPrompt: '',
```

Comment:
Why are we setting customPrompt to be empty?

Reply:
By default, when we add them, they should be empty

Reply:
Even on scheduled page?

Reply:
it's onlt for adding new applications

Reply:
+ we don't get this from api

Reply:
Is there way to save the custom prompt, or one should enter it each time report requested?

Reply:
it's saved when scheduled report is planned. Than when user open this page it would automaticly load